contracts/BoostedMasterChefJoe.sol

```diff
@@ -0,0 +1,621 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.12;
```

Comment:
let's use `0.7.6` to be consistent w/ the other veJOE and sJOE contracts

Reply:
The only issue I see with that is that I started with a base of MasterChefJoeV3 and have just changed some methods but left all the other logic unchanged. Will need to research that the existing part of the contract still works with the updated version.

Reply:
I think either should be fine here. Big question - do we/should we make it upgradeable?

Reply:
No reason other than the original MasterChef wasnt upgradeable.

Reply:
I have changed this to use upgradable proxy.